src/meta/client/src/grpc_client.rs

```diff
@@ -530,7 +530,9 @@ impl MetaGrpcClient {
             let channel = self.make_channel(Some(addr)).await;
             match channel {
                 Ok(c) => {
-                    let mut client = MetaServiceClient::new(c.clone());
+                    let mut client = MetaServiceClient::new(c.clone())
+                        .max_decoding_message_size(16 * 1024 * 1024)
```

Comment:
This arithmetic expression appears many times and is recommended as a constant.

Reply:
Done. my man